tox.ini

```diff
@@ -7,5 +7,5 @@
 [testenv]
 deps=-rtests/requirements.txt
 passenv=APPDATA USERPROFILE HOMEDRIVE HOMEPATH windir APPVEYOR
-commands=py.test {posargs:-rsx}
```

Comment:
The reason I had the arguments here was I wanted them to be readily overwritten. Otherwise, I don't think there's a way to omit or change the reporting. I do see how it's somewhat counter-intuitive when one specifies an argument for another purpose, though. Yes, perhaps it's better the way you've done it.